.github/workflows/test.yml

```diff
@@ -17,4 +17,4 @@ jobs:
     - name: Build
       run: go build -v .
     - name: TF tests
-      run: go test -v -cover -parallel 4 ./...
+      run: go test -v -cover -parallel 4 $(go list ./... |grep -v 'tests')
```

Comment:
We could rename the tests to `TestExampleFoo` or `TestTerragruntFoo` and then use `go test- skip {Pattern}`, but this seems well enough for now.